src/groundStation/system.py

```diff
@@ -1385,7 +1385,24 @@ def __init__(self):
                                 'eNIM2_lux': '>i2',
                                 'ram_avail': '>i2',
                                 'lowest_img_num': '>i2',
-                                'first_blank_img_num': '>i2'
+                                'first_blank_img_num': '>i2',
+                                #IRIS
+                                '###############################\r\n'
+                                'Iris Board\r\n'+
+                                '###############################\r\n'+
+                                'VIS Temperature': '>u2',
+                                'NIR Temperature': '>u2',
+                                'Flash Temperature': '>u2',
+                                'Gate Temperature': '>u2',
+                                'Image number': '>u1',
+                                'Software Version': '>u1',
+                                'Error number': '>u1',
+                                'MAX 5V voltage': '>u2',
+                                'MAX 5V power': '>u2',
+                                'MAX 3V voltage': '>u2',
+                                'MAX 3V power': '>u2',
+                                'MIN 5V voltage': '>u2',
+                                'MIN 3V voltage': '>u2',    
```

Comment:
Noted. Change applied -> ([83fbc1c](https://github.com/AlbertaSat/ex2_ground_station_software/pull/111/commits/83fbc1c5b64482ca379cf3d71b7a9626bf1892ab))

Reply:
Can you also apply this change to lines 1392-1405 with this change as the website side uses `HOUSEKEEPING.GET_HK` to grab all the housekeeping data

Reply:
Sure, I can do that. I actually remember to update the changes in housekeeping service and accidentally updated `CommandsDocs.txt` instead of `system.h` :neutral_face: